hw/bsp/rb-nano2/rb-nano2_download.sh

```diff
@@ -37,6 +37,15 @@ if [ "$MFG_IMAGE" ]; then
     FLASH_OFFSET=0
 fi
 
+# We write the config registers for BPROT so that NVMC write
+# protection gets disabled for all blocks of the flash
+# We also write the DISABLEINDEBUG register so that, write
+# protection gets disabled in debug register by default
+EXTRA_GDB_CMDS="set {unsigned long}0x40000600=0;set {unsigned long}0x40000604=0;\
```

Comment:
Sorry, my bad. I assumed there was a EXTRA_GDB_CMDS in the openocd script. CFG_POST_INIT goes to the OCD_CMD_FILE which is more of ocd commands than gdb commands. I think what is missing in the openocd script is EXTRA_GDB_CMDS which goes to the GDB_CMD_FILE.